src/main/comments-store.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+class CommentsStore {
+  constructor() {
+    this.map = new WeakMap();
+  }
+  addComment(node, comment, type) {
```

Comment:
This can be private?

Reply:
Same as above, not supported

Reply:
Can we use soft private by TypeScript compiler with JSDoc comments?

```js
/**
@private
*/
```